contributors/devel/release/testing.md

```diff
@@ -0,0 +1,141 @@
+# Kubernetes test sustaining engineering
+
+This document describes how Kubernetes automated tests are maintained as part
+of the development process.
+
+## Definitions
+
+- *test*
+  - *artifact*: a row in [test grid]
+  - a single test run as part of a test job
+  - maybe either an e2e test or an integration / unit test
+- *test job*
+  - *artifact*: a tab in [test grid]
+  - a collection of tests that are run together in shared environment. may:
+    - run in a specific environment - e.g. [gce, gke, aws], [cvm, gci]
+    - run under specific conditions - e.g. []upgrade, version skew, soak, serial]
+    - test a specific component - e.g. [federation, node]
+- *test infrastructure*
+  - not directly shown in the test grid
+  - libraries and infrastructure common across tests
+- *test failure*
+  - persistently failing test runs for a given test
+- *test flake*
+  - periodically failling test runs for a given test
+  
+## Ownership
+
+Each test must have an escalation point (email + slack).  The escalation point is responsible for
+keeping the test healthy.  Fixes for test failures caused by areas of ownership outside the
+responsibility of the escalation point should be coordinated with other teams by the
+test escalation point.
+
+Escalation points are expected to be responsive within 24 hours, and prioritize test failure
+issues over other issues.
+
+### test
+
+Each test must have an owning SIG or group that serves as the escalation point for flakes and failures.
+The name of the owner should be present in the test name so that it is displayed in the test grid.
+
+Owners are expected to maintain a dashboard of the tests that they own and
+maintain the test health.
+
+### test job
+
+Each test job must have an owning SIG or group that is responsible for the health of the test job.  The
+owner may also serve as an escalation point for issues impacting a test only in that specific test job
+(passing in other test jobs).  e.g. If a test only fails on aws or only on gke test jobs, the test job
+owner and test owner must identify the owner for resolving the failure.
+
+Owners of test jobs are expected to maintain a dashboard of the test jobs they own and
+maintain the test job health.
+
+### test infrastructure
+
+Issues with underlying test infrastructure (e.g. prow) should be escalated to XXXX.
+(Where are these escalated?)
```

Comment:
One comment is that I would like us to more clearly separate infrastructure like prow (which test-infra-maintainers wants to fix) from environmental things (running out of GCP quota, slowness loading images etc) which we should escalate to the job owners.